### ISSUE TITLE
Extract metrics from slogs and send them to otel

### ISSUE THREAD

**Issue by toliaqat** [Agoric/Agoric#9579]
## What is the Problem Being Solved?
Currently, we process the extensive data in slogs using offline tools such as jq or custom scripts. However, adopting a method similar to the [otel/tracing](https://github.com/Agoric/agoric-sdk/blob/master/packages/telemetry/src/otel-trace.js) could enable us to extract valuable metrics directly from slogs and send them as metrics (otel).

We can focus on extracting two types of metrics:

- Pre-existing Backend Metrics: Specifically from the `cosmic-swingset-after-commit-stats` type slog entry, which includes statistics like forcedGc, memoryUsage, and heapStats.
- Calculated Performance Metrics: Derived from various events, such as the duration it takes to complete a crank, contract call durations, syscall durations, etc.
- This approach would streamline our monitoring process and improve the real-time visibility of our system's performance.

**Comment by mhofman** [comment]
Interesting fields from the slog:
`cosmic-swingset-commit-block-finish`:

-   `saveTime` : swing-store commit
-   `fullSaveTime`: total commit time including cosmos DB

`cosmic-swingset-after-commit-stats`:

-   `memoryUsage.heapUsed`: JS heap used for cosmic-swingset + kernel
-   `statsTime.forcedGc`: time (in ms) taken to do a force gc after block commit

**Comment by mhofman** [comment]
We need to evaluate whether we need to re-implement all cosmic-swingset / kernel explicitly generated metrics before we can switch over, or if it would be possible to report metrics from 2 different places (and processes)

**Comment by gibson042** [comment]
Should this be closed as a duplicate of #11045?

**Comment by mhofman** [comment]
Yeah probably, it has been superseded by #11045 (the framework to report metrics from slog events), and adding metrics as tracked in #10882
